simulation/traffic_simulator/src/entity/entity_base.cpp

```diff
@@ -217,6 +217,37 @@ auto EntityBase::getLaneletPose(double matching_distance) const
   }
 }
 
+auto EntityBase::refillEntityStatusWithLaneletData(
```

Comment:
How about rename to `fillLaneletPose` ?

Reply:
I've renamed and moved to `EntityManager` class -> https://github.com/tier4/scenario_simulator_v2/commit/70b2f45bc2993f718afa77bd367fd7d9ec2bd10d